### ISSUE TITLE
can't get location in China

### ISSUE THREAD

**Issue by superau** [Baseflow/flutter-geolocator#438]
## 🔙 Regression

Hi, i am in China, but i can use google service and google map normally with vpn.

I want to get and print location from this pcakage, but nothing.

### Code
```
@override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            if (_currentPosition != null)
            Text(
                "LAT: ${_currentPosition.latitude}, LNG: ${_currentPosition.longitude}"),
            RaisedButton(
              onPressed: () {
                //Get the current location
                getLocation();
              },
              child: Text('Get Location'),
            ),
          ],
        ),
      ),
    );
  }

  void getLocation() async {
    print('start');
    final Geolocator geolocator = Geolocator()..forceAndroidLocationManager;

    geolocator
        .getCurrentPosition(desiredAccuracy: LocationAccuracy.lowest)
        .then((Position position) {
          print(position);
      setState(() {
        _currentPosition = position;
      });
    }).catchError((e) {
      print(e);
    });
  }
}

```

### Configuration
```
D:\workspace\Clima-Flutter>flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, v1.12.13+hotfix.9, on Microsoft Windows [Version 10.0.18363.836], locale zh-CN)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[√] Android Studio (version 3.6)
[!] IntelliJ IDEA Ultimate Edition (version 2019.3)
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
[√] Connected device (1 available)

! Doctor found issues in 1 category.

```
### physical device
name: XiaoMi RedMi K20
android version: 10
MIUI: 11.0.6  

### Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android

**Comment by mvanbeusekom** [comment]
This issue should be solved with version 6.0.0 (you can try version 6.0.0-rc.4 if you would like to verify)

**Comment by fluttercorner** [comment]
[How to Get Device Lat Long in Flutter](https://fluttercorner.com/how-to-get-device-lat-long-in-flutter/)
